File: fixtures/ts-project/calculator.ts
import { formatResult, add } from './utils';

export class Calculator {
  private history: string[] = [];

  compute(n: number): string {
    const result = formatResult(n);
    this.history.push(result);
    return result;
  }

  getHistory(): string[] {
    return this.history;
  }

  sum(a: number, b: number): number {
    return add(a, b);
  }
}

export function runCalculator(n: number): void {
  const calc = new Calculator();
  console.log(calc.compute(n));
}

File: fixtures/python-project/calculator.py
from math_utils import format_result, add

class Calculator:
    def __init__(self):
        self.history = []

    def compute(self, n):
        result = format_result(n)
        self.history.append(result)
        return result

    def get_history(self):
        return self.history

    def sum(self, a, b):
        return add(a, b)

def run_calculator(n):
    calc = Calculator()
    print(calc.compute(n))
